netbox/virtualization/views.py

```diff
@@ -172,7 +173,17 @@ class ClusterView(generic.ObjectView):
     queryset = Cluster.objects.all()
 
     def get_extra_context(self, request, instance):
-        return instance.virtual_machines.aggregate(vcpus_sum=Sum('vcpus'), memory_sum=Sum('memory'), disk_sum=Sum('disk'))
```

Comment:
We need to keep the aggregation logic at the database layer in the interest of performance.